tests/cypress/e2e/features/ground_truth_jobs.js

```diff
@@ -397,4 +415,63 @@ context('Ground truth jobs', () => {
             cy.checkFrameNum(groundTruthFrames[3]);
         });
     });
+
+    describe('Testing case ground truth job list', () => {
+        const imagesCount = 60;
+        const imageFileName = 'ground_truth_2';
+        const width = 100;
+        const height = 100;
+        const posX = 10;
+        const posY = 10;
+        const color = 'gray';
+        const archiveName = `${imageFileName}.zip`;
+        const archivePath = `cypress/fixtures/${archiveName}`;
+        const imagesFolder = `cypress/fixtures/${imageFileName}`;
+        const directoryToArchive = imagesFolder;
+
+        before(() => {
+            cy.visit('/tasks');
+            cy.imageGenerator(imagesFolder, imageFileName, width, height, color, posX, posY, labelName, imagesCount);
+            cy.createZipArchive(directoryToArchive, archivePath);
+            cy.createAnnotationTask(
+                taskName, labelName, attrName,
+                textDefaultValue, archiveName, false,
+                { multiJobs: true, segmentSize: 2 },
+            );
```

Comment:
if possible I would propose to use headless creating

Reply:
As I remember there was a problem with attaching files to cypress requests, so for now ill stick to manual task creation